docs/technical_reference/edb/index.rst

```diff
@@ -9,17 +9,17 @@ Electrolyte Database (EDB)
 
 Overview
 --------
-The Electrolyte Database (EDB) stores metadata and data about chemical species, called here 
+The Electrolyte Database (EDB) stores metadata and data about chemical species, called here
 `components`, and `reactions`. It is accessed through a Python API to return well-defined Python objects.
 
-The data are stored in `MongoDB <https://mongodb.org>`_, so they can be queried in a number of ways, and the 
-system is extensible to new use-cases. The native storage format for MongoDB is a `JSON <https://json.org>`_ document, 
-and the expected structure and fields of the *component* and *reaction* data is defined by a 
+The data are stored in `MongoDB <https://mongodb.org>`_, so they can be queried in a number of ways, and the
+system is extensible to new use-cases. The native storage format for MongoDB is a `JSON <https://json.org>`_ document,
+and the expected structure and fields of the *component* and *reaction* data is defined by a
 `JSON Schema <https://json-schema.org>`_. Validation using those schemas is built into the API (though it can be disabled).
 
 To interface with the `IDAES Core Modeling Framework <https://idaes-pse.readthedocs.io/en/stable/user_guide/concepts.html>`_
-(IDAES-CMF, which underlies WaterTAP), add components and reactions to a "base" object and fetch the result as a Python 
-`dict`. This result can be used to configure and build IDAES objects (`ParameterBlocks`, `ReactionBlocks`, etc.). 
+(IDAES-CMF, which underlies WaterTAP), add components and reactions to a "base" object and fetch the result as a Python
+`dict`. This result can be used to configure and build IDAES objects (`ParameterBlocks`, `ReactionBlocks`, etc.).
 The API also has methods to construct component and reaction objects from IDAES configurations.
 
```

Comment:
I would add a link to the "how-to" docs (and examples, if there are any, I forget) here